pkg/base/constants.go

```diff
@@ -41,6 +41,10 @@ const (
 
 	// KubeletRootPath is the pods' path on the node
 	KubeletRootPath = "/var/lib/kubelet/pods"
+
+	// OpenShiftHostRootPath is root mount in pods for OpenShift cluster
+	OpenShiftHostRootPath = "/hostroot/sysroot"
```

Comment:
I prefer to avoid having names of particular platforms in the code.

Reply:
Fixed